src/main/resources/templates/home.html

```diff
@@ -39,11 +39,13 @@ <h1>
     <h3><a href="/user">Users
     </a><br>
         <a href="/login">New user login
-        </a>
+        </a><br>
         <a href="/register">New user registration
-        </a>
+        </a><br>
         <a href="/logout">Logout
-        </a>
+        </a><br>
+        <a href="/modifyLoggedUser">Change your data
```

Comment:
Nie masz takiego endpointa na backendzie - masz tylko /modifyLoggedUser/{id} - więc raczej chciałabyś albo przekazać tutaj id, albo stworzyć nowego endpointa. Inaczej to nie zadziała